### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by emredogan7**
Updated README.md file and added necessary changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # pr_observation_repo
 just a trial.
+
```
Comment (trial-user-mrdogan7) [inline README.md:3]: unnecessary new line. Can you remove it?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+Creating a new pull request.
```
Comment (trial-user-mrdogan7) [inline README.md:4]: can you give more details?

File: README.md (hunk 1 above)
Comment (emredogan7) [reply README.md:3]: It is necessary for bla bla.

File: README.md (hunk 2 above)
Comment (emredogan7) [reply README.md:4]: okey, i added the required details.

Comment (trial-user-mrdogan7) [reply README.md:4]: LGTM.

**Review by trial-user-mrdogan7 — APPROVED** [review]
All required feedbacks completed. Ready to push.

**Pull request merged**